### PR TITLE
Fix issue#15

### PR DESCRIPTION
**Pull request by bz-liu**
Fix issue #15

### CODE REVIEW

File: src/PointerAnalysis/Util/GraphWriter.h [hunk 1]
```diff
@@ -177,13 +177,14 @@ class GraphWriter {
     // Output all of the edges now
     edge_iterator EI = GTraits::child_edge_begin(Node);
     edge_iterator EE = GTraits::child_edge_end(Node);
+    // bz: seems like the 2nd param of writeEdge will not exceed max int
     for (unsigned i = 0; EI != EE && i != 64; ++EI, ++i)
```
Comment (BradSwain) [inline src/PointerAnalysis/Util/GraphWriter.h:181]: can `i` be changed to `int` as well?

Comment (bz-liu) [reply src/PointerAnalysis/Util/GraphWriter.h:181]: ```i``` is from type ```edge_iterator``` which we iterate on on line 181 and is returned by llvm code (```llvm::GraphTraits<GraphType>```).

Comment (bz-liu) [reply src/PointerAnalysis/Util/GraphWriter.h:181]: will update.

**Review by BradSwain — APPROVED** [review]

**Comment by BradSwain** [comment]
fixes #15

**Pull request merged**